crates/extensions/c8y_mapper_ext/src/lib.rs

```diff
@@ -14,6 +14,8 @@ pub mod service_monitor;
 #[cfg(test)]
 mod tests;
 
+pub(crate) const C8Y_BRIDGE_TOPIC_ID: &str = "device/main/service/mosquitto-c8y-bridge";
```

Comment:
We can't assume this hardcoded `device/main` topic id even for the main device since we support overriding `device_topic_id` via the config. That being said, I'm not fully sure if all tedge components properly respect that config setting yet either. I'm fairly sure that it is still hardcoded in many places.